db/seeds.rb

```diff
@@ -1,7 +1,43 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+# Create users
+5.times do
+  user = User.create!(
+    email: Faker::Internet.email,
+    password: Faker::Internet.password
+  )
+  user.confirm
```

Comment:
You can also skip confirmation. I've used both approaches. Confirming the users is a little more expensive (in terms of database/computation time.